CHANGES.rst

```diff
@@ -1,5 +1,24 @@
 .. Red changelogs
 
+Redbot 3.5.7 (2024-03-24)
+=========================
+
+| Thanks to all these amazing people that contributed to this release:
+| :ghuser:`aikaterna`, :ghuser:`Flame442`, :ghuser:`karlsbjorn`, :ghuser:`Jackenmen`
+
+This is a hotfix release fixing a bug with Red's reaction-based menus introduced in the previous release.
+
+End-user changelog
+------------------
+
+Fixes
+*****
+
+- **Core** - Fixed an issue with Red's reaction-based menus *with custom controls* not working properly (:issue:`6324`)
+- **Core - Bot Commands** - Updated supported image formats in ``[p]set bot avatar``'s error messages to include GIFs (:issue:`6323`)
+
+----
```

Comment:
I don't think this should be here, it renders as a little line in the docs that isn't there for any other entry.

Reply:
You probably looked at the version built by RTD before the [`765e407` (#6325)](https://github.com/Cog-Creators/Red-DiscordBot/pull/6325/commits/765e407b7bd4db3aca481a39971e8a4e27f6ec23) commit. Try refreshing.